### ISSUE TITLE
GL context creation failed

### ISSUE THREAD

**Issue by DuckerMan** [sebcrozet/kiss3d#129]
I try to compile example:

`thread 'main' panicked at 'called `Result::unwrap()` on an `Err` value: OsError("GL context creation failed")', libcore/result.rs:945:5
note: Run with `RUST_BACKTRACE=1` for a backtrace.
`

**Comment by martinellison** [comment]
I am getting the same error now (when my app tries to open its window). My apps used to work but I upgraded everything (so I'm not sure whether this is a Mesa issue or a kiss3d issue). glxgears is fine. I'm not sure how to debug this; any advice?

**Comment by sebcrozet** [comment]
@martinellison Could you try to run one of the examples of kiss3d and see if it fails? For example you may run `cargo run --example cube` from kiss3d's root directory.

**Comment by DuckerMan** [comment]
Yes, it fails. I guess it hapenned because of my GPU. it supports only OpenGL 2.1
